dune

```diff
@@ -0,0 +1,20 @@
+(env
+ (gprof
+  (ocamlopt_flags
+   (:standard -g -p -w -39)
+  )
+  (flags
+   (:standard -w -39)
+  )
+ )
+ (dev
+  (flags
+   (:standard -g -w -39)
+  )
+ )
+ (release
+  (flags
+   (:standard -w -39)
```

Comment:
Are you sure you want to get rid of `g` here? The code may become undebuggable

Reply:
-g should be part of :standard